### PR TITLE
dev-libs/protobuf: version bump, 3.0.0_beta2

### PR DESCRIPTION
**Pull request by tomboy-64**
This version provides support for proto3, the new protobuffer format. It's also a bump to so.10.

Further changes to the ebuild include:
- bump to EAPI6
- enabled python 3.{3,4,5}
- enabled Java "nano" binding

@vapier 
@gentoo/java
@gentoo/python
@gentoo/proxy-maint

### CODE REVIEW

File: dev-libs/protobuf/protobuf-3.0.0_beta2.ebuild [hunk 1]
```diff
@@ -0,0 +1,174 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+JAVA_PKG_IUSE="doc source"
+PYTHON_COMPAT=( python2_7 python3_3 python3_4 python3_5 ) # can't go newer cause of google-apputils
```
Comment (rindeal) [inline dev-libs/protobuf/protobuf-3.0.0_beta2.ebuild]: could be simplified with `python3_{3,4,5}`

**Comment by tomboy-64** [comment]
It also still lacks working untittests. Assistance with this is welcome.

**Comment by tomboy-64** [comment]
also, cross-compilation is still botched

**Comment by tomboy-64** [comment]
going to close this for the time being, as it needs more work regarding tests and cross-compilation.

**Pull request closed without merging**